server/app/app.js

```diff
@@ -49,6 +49,17 @@ const updateWishlist = async (id, { name, author }) => {
   });
 };
 
+const updateItem = async (id, { name }) => {
+  return new WishListItem({ id, name }).save(null, {
+    require: true,
+    method: 'update',
+  });
```

Comment:
This should include `patch: true`. Refer to the [Bookshelf documentation for save](https://bookshelfjs.org/api.html#Model-instance-save)

Reply:
The relevant part of the documentation:
> - [patch=false] Boolean
Only save attributes supplied as arguments to the save call, ignoring any attributes that may be already set on the model.